library/src/main/java/com/datatorrent/lib/fileaccess/FileAccess.java

```diff
@@ -151,7 +151,7 @@
      * @param value
      * @throws IOException
      */
-    void append(byte[] key, byte[] value) throws IOException;
+    void append(Slice key, Slice value) throws IOException;
```

Comment:
Yes, but it would make existing subclasses not compilable. 
Anyway, my point is that this change does not actually affect anybody in reality.